lib/normalize-name.spec.js

```diff
@@ -0,0 +1,26 @@
+var normalize = require("./normalize-name");
+
+describe("normalize", () => {
```

Comment:
I actually started a `add-tests` branch with a test for this with `describe.each`: https://github.com/Financial-Times/n-heroku-tools/blob/533857c744528889d25db2de8ccedde619767c5c/lib/normalize-name.test.js